src/index.ts

```diff
@@ -215,7 +216,8 @@ export async function isAvailable() {
     }
     await cachedIsAvailableResponse;
     return true;
-  } catch (err) {
+  } catch (e) {
+    const err = e as GaxiosError & {type: string};
```

Comment:
Is there a cleaner way to accomplish this? It looks like e is a `GaxiosError` with a possible type field but couldn't find a concrete type definition that matches.